pages/business/setup/download-key.tsx

```diff
@@ -0,0 +1,160 @@
+import * as React from 'react'
```

Comment:
was it necessary to rename this file? It makes it difficult to review the changes.
The purpose of this page is still to verify the key.

Reply:
no it was not. However the purpose changed. Before that "verify-key" was correct because the page before that triggered the download (which didn't work on iOS). So now the purpose is download and verify. Anyway I renamed it back so the diff gets easier to read